test/test-1155-to-20.js

```diff
@@ -58,6 +58,9 @@ contract('Wrapped1155Factory', function (accounts) {
   });
   
   let unusedId, singleId, batchIds;
+  let tokenName = web3.utils.utf8ToHex("WrappedERC-1155");
```

Comment:
Should we use `wERC-1155` to have more space for custom names (as the limit is 31bytes)?

Reply:
This was replaced with encoded values using [1155-to-20-helper](github.com/davidalbela/1155-to-20-helper)